scripts/examples/nrfconnect_example.sh

```diff
@@ -30,8 +33,13 @@ if [[ ! -f "$APP/nrfconnect/CMakeLists.txt" ]]; then
     exit 1
 fi
 
+if [ -z "$BOARD" ]; then
+    echo "No mandatory BOARD argument supplied!"
+    exit 1
+fi
+
 set -x
 [[ -n $ZEPHYR_BASE ]] && source "$ZEPHYR_BASE/zephyr-env.sh"
 env
 
-west build -b nrf52840dk_nrf52840 -d "$APP/nrfconnect/build" "$APP/nrfconnect"
+west build -b $BOARD -d "$APP/nrfconnect/build" "$APP/nrfconnect" -- "$@"
```

Comment:
This should fix the restyled error.

```suggestion
west build -b "$BOARD" -d "$APP/nrfconnect/build" "$APP/nrfconnect" -- "$@"
```

Reply:
Great, that did the job, thanks!